ui/src/main/webapp/src/app/executions/executions-list.component.html

```diff
@@ -30,7 +30,16 @@
         </thead>
         <tbody>
         <tr *ngFor="let execution of sortedExecutions" [class]="getClass(execution)" class="execution-row">
-            <td><a [routerLink]="['/projects', execution.projectId, execution.id, 'execution-details']">#{{execution.id}}</a></td>
+            <td>
+                <span *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports; else elseExecutionState">
+                    <a  class="pointer link" target="_blank" href="{{formatStaticReportUrl(execution)}}" title="Show reports">
+                        #{{execution.id}} <i class="fa fa-external-link fa-fw"></i>
```

Comment:
I can try to move the icon and upload a screenshot to see if it looks better.
I placed the `external-link` icon after the analysis id just for consistency with the report itself that has that icon always after the link it's referred to.